querying/github.go

```diff
@@ -167,6 +173,24 @@ func (gh *GithubDataSource) processRepoFindings(projects *ProjectCollection, rep
 			finding.Severity = githubSeverities[vuln.SecurityVulnerability.Severity]
 		}()
 	}
+
+	if repo.VulnerabilityAlerts.PageInfo.HasNextPage {
+		var repositoryQuery repositoryQuery
```

Comment:
Careful! It looks like you are unintentionally shadowing this name here -- using the same name in an inner scope as in an outer scope. This can potentially cause issues! I would recommend using a different name for this variable, something like

```suggestion
		var repoQuery repositoryQuery
```
(Also updating line 184-191 to reflect this)
This ensures there will be no conflicts, confusion, or weirdness.

Reply:
Fixed